lib/data-types.js

```diff
@@ -358,7 +358,7 @@ class DECIMAL extends NUMBER {
   }
   toSql() {
     if (this._precision || this._scale) {
-      return `DECIMAL(${[this._precision, this._scale].filter(_.identity).join(',')})`;
```

Comment:
`identity` is clear

Reply:
Actually, what if `_scale` is specified but `_precision` isn't, wouldn't that generate a bad result?

Reply:
Yes, it will. I think many database allow just precision, so I think that is what being handled here. In worst case you can specify just the `scale`.

But I think those errors should be caught in development phase. Proper validation for this can be implemented when construing DECIMAL though